hackportal.config.ts

```diff
@@ -557,6 +558,13 @@ interface datalistQuestion extends RegistrationQuestion {
   datalist: string;
 }
 
+export type DateRange = { start: Date; end: Date };
+
+interface AvailabilityInfoQuestion extends RegistrationQuestion {
+  value: DateRange[];
+  range: DateRange;
+}
+
```

Comment:
Is there any rationale behind having these types in the config file instead of in `@generated/types` ?